qiskit/result/result.py

```diff
@@ -230,7 +230,12 @@ def get_memory(self, experiment=None):
                 raise QiskitError('Measurement level {} is not supported'.format(meas_level))
 
         except KeyError:
-            raise QiskitError('No memory for experiment "{}".'.format(experiment))
+            raise QiskitError(
+                'No memory for experiment "{}". '
+                'Please verify that you either ran a measurement level 2 job '
+                'with the memory flag set, eg., "memory=True", '
+                'or a measurement level 0/1 job.'.format(experiment)
```

Comment:
Are these measurement levels standard terminology across multiple backends or just an ibmq thing? I ask because on other backends I'm wondering if this actually means anything, especially because the bug this is referencing was originally opened using BasicAer (it was edited later to change the backend in the example) not ibmq where measurement level doesn't mean anything.

Reply:
These are standard arguments to [execute](https://qiskit.org/documentation/apidoc/execute.html).

Reply:
Sure, but `meas_level` is what I'm mildly concerned about. It's only a kwarg on execute because of qobj and assembly right? Nothing besides an IBMQ backend will support it. Like if you set `meas_level=2` and you're using the aqt provider or basic aer it won't do the right thing it will just ignore that parameter (although aqt will raise a warning because it's on the new providers interface).

Reply:
`meas_level` is a required input for any pulse backend execution.